src/Features/Core/Portable/PullMemberUp/AbstractPullMemberUpRefactoringProvider.cs

```diff
@@ -65,7 +70,15 @@ public override async Task ComputeRefactoringsAsync(CodeRefactoringContext conte
                 return;
             }
 
-            PullMemberUpViaQuickAction(context, selectedMember, allDestinations);
+            var allActions = allDestinations.SelectAsArray(
+                destination => MembersPuller.Instance.TryComputeCodeAction(context.Document, selectedMember, destination)).
+                WhereAsArray(action => action != null).
+                Concat(new PullMemberUpWithDialogCodeAction(context.Document, selectedMember, this));
+
+            var nestedCodeAction = new CodeActionWithNestedActions(
+                string.Format(FeaturesResources.Pull_0_up, selectedMember.ToNameDisplayString()),
+                allActions, allActions.Length < 5);
```

Comment:
Why do you need the `allActions.Length < 5`?  Why not always mark this as inlinable and defer to the default behavior of the normal smart lightbulb?